src/assets/style/module/changelog/_changelog.less

```diff
@@ -0,0 +1,97 @@
+.changelog-container {
+    width: 480px;
+    max-height: 480px;
+    overflow-y: scroll;
+    margin-left: -240px;
+    padding: 12px 15px;
+    position: absolute;
+    top: 20%;
+    left: 50%;
+    background-color: @dark;
+    color: @light-gray;
+    font-size: 12px;
+    font-family: inherit;
+    opacity: 0;
+    border-radius: 1px;
+    box-sizing: border-box;
+    box-shadow: 0 0 0 6px @blue-medium;
+    pointer-events: none;
+    transition: opacity 0.5s ease;
+    z-index: 200;
+}
+
+.changelog-container.changelog-container-is-visible {
```

Comment:
just stick with the single class declarations here and below.  `.changelog-container` can be removed from all of the following items